src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star star--1"></div>
+
+      <div class="stars__star star--2"></div>
+
+      <div class="stars__star star--3"></div>
+
+      <div class="stars__star star--4"></div>
+
+      <div class="stars__star star--5"></div>
+    </div>
```

Comment:
```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
```
Stars elements shouldn't have any additional classes. 
Also, you can remove empty lines to improve readability as divs are empty
![image](https://github.com/mate-academy/layout_stars/assets/96409563/c45bc8f9-b330-4a3a-b400-8e96959e8d44)